centralserver/i18n/management/commands/generate_dubbed_video_mappings.py

```diff
@@ -50,6 +50,8 @@ def download_ka_dubbed_video_csv(download_url=None, cache_filepath=None):
         else:
             download_url = download_url.replace("/edit", "/export?format=csv")
 
+    download_url = "https://docs.google.com/spreadsheets/d/1k5xh2UXV3EchRHnYzeP6-YGKrmba22vsltGSuU9bL88/edit#gid=0"
```

Comment:
Hmm, it's called "APRIL 2015 MASTER LIST" -- is it permanent in the sense that they stopped updating after that, due to deprecating the spreadsheet approach in favor of the API?